### PR TITLE
acrn-config: modify board info of block device info

### PR DESCRIPTION
**Pull request by liuwei987**
As the blkid info contains 'UUID' information, which easy to lead to
confuse with the vm 'UUID'. So refine the format for block device.

v1-v2:
    1). align the style in BLOCK_DEVICE_INFO tag
    2). add 'mmcblk1p3' in BLOCK_DEVICE_INFO tag

v2-v3:
    1). correct the commit message.

Tracked-On: #3754
Signed-off-by: Wei Liu <weix.w.liu@intel.com>
Acked-by: Victor Sun <victor.sun@intel.com>

### CODE REVIEW

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

File: misc/acrn-config/target/parser_lib.py [hunk 1]
```diff
@@ -83,13 +83,20 @@ def cmd_execute(cmd):
...
         if "ext4" in root_type or "ext3" in root_type:
-            return True
+            block_type = ''
+            block_dev = line.split()[0]
+            for type_str in line.split():
+                if "TYPE=" in type_str:
+                    block_type = type_str
+
+            block_format = block_dev + " " + block_type
+            return block_format
```
Comment (binbinwu1) [inline misc/acrn-config/target/parser_lib.py]: Can python return value be different types?

Comment (liuwei987) [reply misc/acrn-config/target/parser_lib.py]: The python can support it. But it is not the good coding style. I will update it.
Thanks.

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Pull request merged**